chcemvediet/apps/anonymization/models.py

```diff
@@ -196,6 +196,98 @@ def save(self, *args, **kwargs):
     def __unicode__(self):
         return format(self.pk)
 
+class AttachmentAnonymizationQuerySet(QuerySet):
+    def order_by_pk(self):
+        return self.order_by(u'pk')
+
+class AttachmentAnonymization(FormatMixin, models.Model):
+
+    # May NOT be NULL
+    attachment = models.ForeignKey(Attachment)
+
+    # May NOT be NULL
+    successful = models.BooleanField(default=False,
+            help_text=squeeze(u"""
+                True if anonymization has succeeded, False otherwise.
+                """))
+
+    # Filename may be empty; Random local filename is generated in save() when creating a new object
+    file = models.FileField(upload_to=u'attachment_annonymizations', max_length=255, blank=True,
+            help_text=squeeze(u"""
+                Empty filename if anonymization failed.
+                """))
+
+    # May be empty: Extension automatically adjusted in save() when creating new object.
+    name = models.CharField(max_length=255, blank=True,
+            help_text=squeeze(u"""
+                Attachment anonymization file name, e.g. "document.pdf". Extension automatically
```

Comment:
Premenovat `"document.pdf"` na `"document.odt"`, kedze v modeli `AttachmentsAnonymization` ocakavame `.odt` subory. Analogicky aj v `AttachmentRecognition`? Ak ano, tieto veci by som upravil v PR, kde chceme celu dokumataciu prepisat do markdown.

Reply:
Ano. Dobry napad, example by mali byt realisticke. Takze ano, tam kde to budu `.odt` subory je dobre dat ako priklad `document.odt`.

Kedze to meni model a aj migraciu, zmenu sprav v inom PR nez prepisovanie do markdown. PR s prepisovanim do MD by nemal menit obsah iba formu. Kludne samostatny PR.

Reply:
:ok: